src/CMakeLists.txt

```diff
@@ -18,7 +18,10 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DIN_NGRAPH_LIBRARY")
 
 include_directories(ngraph)
 
-add_subdirectory(resource)
+if (NOT NGRAPH_DEX_ONLY)
```

Comment:
Remove this check. Always add resource. If it is not needed then it won't be built.